src/containers/BadgeContainer.jsx

```diff
@@ -18,7 +18,7 @@ class BadgeContainer extends React.Component {
   constructor() {
     super();
     this.state = {
-      caesarData: [],
+      caesarData: new Map(),
```

Comment:
Be careful with this in IE11, which doesn't support `map.values()`.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map#Browser_compatibility

Reply:
Thank you for the heads up! I'll check with the project lead, keep an eye out for, and see if I can discern from Google Analytics if it'll be an issue.